ui/components/component-library/icon/icon.test.js

```diff
@@ -2,21 +2,13 @@
 import { render } from '@testing-library/react';
 import React from 'react';
 import { SIZES, COLORS } from '../../../helpers/constants/design-system';
+import { ICON_NAMES } from './icon.constants';
 import { Icon } from './icon';
 
-// Icon names are stored in the ICON_NAMES environment variable
-// mocking the environment variable here
-const MOCK_ICON_NAMES = {
-  ADD_SQUARE_FILLED: 'add-square-filled',
-  BANK_FILLED: 'bank-filled',
-  BOOKMARK_FILLED: 'bookmark-filled',
-  CALCULATOR_FILLED: 'calculator-filled',
-};
-
```

Comment:
No longer need to mock the icon names env var because we are setting it in `test/env.js`